markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -94,14 +96,21 @@ contract PerpsAccountModule is IPerpsAccountModule {
 
     // 1. call depositMarketUsd and deposit amount directly to core system
     // 2. look up account and reduce debt by amount
-    // 3b. quoteUnwrap() -> inchQuote -> returnAmount
     function payDebt(uint128 accountId, uint256 amount) external override {
+        FeatureFlag.ensureAccessToFeature(Flags.PERPS_SYSTEM);
         Account.exists(accountId);
+
```

Comment:
Another recommended validation we had for BFP was to ensure no orders were pending for the account. This was to prevent potential gaming with respect to delaying orders and then allowing them to go through via manipulating debt in a single transaction.

Finding for BFP: https://www.notion.so/guardianaudits/M-02-0b54ea9c1dfe439b8bd9ecc3c39ac8fc?pvs=4

The BFP finding leveraged the merge feature, which doesn't exist here, but it still may be a good validation to consider to cut off any other potential manipulations.

https://github.com/Synthetixio/synthetix-v3/pull/2216/files#diff-f2bde65b1d25fef9e34a2acc04220ffc7cdcb2d3c8e28e1dc9ac92396f45cc8eR507